spec/my_enumerables_spec.rb

```diff
@@ -0,0 +1,226 @@
+require './my_enumerables.rb'
+
+describe Enumerable do
+  context '#my_each' do
+    it 'returns original array when no output' do
+      expect([1, 2, 3].my_each { |i| i * 2 }).to eq [1, 2, 3]
+    end
+
+    it 'prints result to stdout from block calculation i * 2' do
+      expect { [1, 2, 3].my_each { |i| print i * 2 } }.to output(/246/).to_stdout
+    end
+
+    it 'returns an Enumerator when no block given' do
+      expect([1, 2, 3].my_each.is_a?(Enumerator)).to be true
+    end
+
+    it 'all above-mentioned tests match the official enumerable method counterpart' do
+      expect([1, 2, 3].my_each { |i| i * 2 }).to eq([1, 2, 3].each { |i| i * 2 })
+      expect { [1, 2, 3].each { |i, index| print i * 2 } }.to output(/246/).to_stdout
```

Comment:
Here you are only testing the original `each` method. This test case is not necessary

Reply:
I'm referring to line 19 only

Reply:
Please refer to Commit No. #6